examples/stable_diffusion_v2/cldm/cldm.py

```diff
@@ -60,6 +60,77 @@ def construct(self, x, timesteps=None, context=None, control=None, only_mid_cont
         return self.out(h)
 
 
+class ControlnetUnetModel(UNetModel):
+    def __init__(self, control_stage_config, guess_mode=False, strength=1.0, *args, **kwargs):
```

Comment:
Why not just edit ControledUnetModel?

Reply:
Because it will influence original inference pipeline. They have a large difference